dnf/plugin.py

```diff
@@ -151,12 +147,13 @@ def _import_modules(package, py_files):
             logger.log(dnf.logging.SUBDEBUG, '', exc_info=True)
 
 
-def _iter_py_files(paths, skips):
+def _iter_py_files(paths, skips, enable_plugins):
     for p in paths:
         for fn in glob.glob('%s/*.py' % p):
             (name, _) = os.path.splitext(os.path.basename(fn))
             if any(fnmatch.fnmatch(name, pattern) for pattern in skips):
```

Comment:
You could use itertools here to group 2 lists instead of doing 2 ifs, but even now it's better than nothing.